### ISSUE TITLE
Add to FlatHub

### ISSUE THREAD

**Issue by szkrd** [yakyak/yakyak#948]
I'm not sure how I can install a standalone flatpak file - everything points towards the [FlatHub](https://github.com/flathub/flathub/wiki/App-Submission) repository, where flatpaks may be published. 

Could you add the app to their repository? The repository is meant to be used by the developer him- or herself, so I'm not sure if it would be good to add it myself. Thank you!

**Comment by FFDA** [comment]
What kind of issues do you having while installing yakyak using flatpak?

**Comment by szkrd** [comment]
I couldn't find a commend to install it directly (like a deb),  but I may have overlooked something.

**Comment by FFDA** [comment]
What way did you tried to install yakyak flatpak? Using GUI or terminal? I suggest you to use terminal, because when I tried to install yakyak's flatpak it failed me the GUI way, but when I tried install it using terminal command (I think in root) it worked. 
So just download the file and open folder that contains the flatpak file in the terminal and use command:
`flatpak install filename`
Just change the _filename_ to the name of the flatpak's name. If it fails it might be that it needs root privileges. Add sudo in front of that command.
